src/axom/quest/tests/quest_initialize.cpp

```diff
@@ -72,6 +75,76 @@ TEST(quest_initialize, signed_distance_pointer_initialize)
   delete input_mesh;
 }
 
+// Test allocation/reallocation using axom::allocate and axom::reallocate bytes
```

Comment:
Thanks for adding these reproducers @gunney1 

Please add some comments to this file about why these tests are here, rather than in `sidre` tests of `core` tests, since it is not obvious.  Specifically, that these tests are reproducers for bugs that we found w/ a static config of umpire mixed with a shared config of axom, and that the failures only appear when there are several shared libs that include the static umpire, e.g. `axom::core`, `axom::sidre`, `axom::mint` and `axom::quest`).

Reply:
Done.